tests/test_ptc.py

```diff
@@ -88,6 +90,62 @@ def setUp(self):
             self.dataset.rawExpTimes[ampName] = timeVec
             self.dataset.rawMeans[ampName] = muVec
 
+    def test_ptcDataset(self):
```

Comment:
Can this be moved to `ip_isr`?  This tests the integrity of the calibration definition, not the measurement code.